### PR TITLE
Clean i os docs

### PR DESCRIPTION
**Pull request by joepadmiraal**

### CODE REVIEW

File: docs/ios-sdk.md [hunk 1]
```diff
@@ -99,17 +94,29 @@ func play() {
...
+Your application needs to call the corresponding player methods for each iOS Application LifeCycle callback
+
+```swift
+public func onApplicationDidBecomeActive()
+public func onApplicationWillResignActive()
+public func onApplicationDidEnterBackground()
+public func onApplicationWillEnterForeground()
+```
+
 ### Livery SDK
```
Comment (joepadmiraal) [inline docs/ios-sdk.md:112]: Livery SDK is now part of the Life Cycle chapter.
Maybe we can make it a ## and name it something like Advanced?

File: docs/ios-sdk.md [hunk 2]
```diff
@@ -203,75 +206,27 @@ liveSDK.audioSessionSettings?.options = [.mixWithOthers]
...
+| `mute`             | `Boolean`                      | `false`    | Determines whether mute button should be displayed or not on the [`Player Controls`](#player-controls).           |
+| `muted`            | `Boolean`                      | `false`    | Determines whether media should be muted or not.                                                                  |
+| `poster`           | `String`                       | `""`       | URL for an image to be shown while the video is loading.                                                          |
+| `quality`          | `Boolean`                      | `false`    | Determines whether select quality button should be displayed or not on the [`Player Controls`](#player-controls). |
+| `sources`          | `String[]`                     | `[]`       | Array of media source URLs from which the first that can be played will be selected.                              |
+| `targetLatency`    | `Integer`                      | `3`        | Target live latency in seconds. If 0 then syncing is disabled.                                                    |
 
 ### Source Protocols
```
Comment (joepadmiraal) [inline docs/ios-sdk.md:231]: I think we can remove the `Source Protocols` chapter. It's not something the customers can change or use.

File: docs/ios-sdk.md [hunk 3]
```diff
@@ -434,31 +389,10 @@ The player will receive the millisecond precision server time from the time urls
 
 In other words, IOS SDK players behave always in sync once initialized, meaning if audio packages or video frames are received later then their presentation timestamps due the network connections, they will be ignored or played back in a faster rate by the player.
 
-## ABR (Adaptive BitRate)
-
-Current version players just select the highest quality video for ABR
-
 ## Error Handling
 
 To be updated with recovery options implementation in v 1.0.0
```
Comment (joepadmiraal) [inline docs/ios-sdk.md:394]: Is this still something we need to do or can improve?
If so, could you create a ticket for it?

File: docs/ios-sdk.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Livery iOS SDK
 
-Ex Machina Group Livery iOS SDK, published as CocoaPods spec on Github: [livery-sdk-ios-podspec](https://github.com/exmg/livery-sdk-ios-podspec).
+Livery iOS SDK, published as CocoaPods spec on Github: [livery-sdk-ios-podspec](https://github.com/exmg/livery-sdk-ios-podspec).
 
 Documentation can be found at: [docs.liveryvideo.com/ios-sdk](https://docs.liveryvideo.com/ios-sdk).
```
Comment (joepadmiraal) [inline docs/ios-sdk.md:6]: The Livery iOS SDK is compatible with iOS 12 **of** higher.

I think that should be iOS 12 **or** higher right?

File: docs/ios-sdk.md (hunk 1 above)
Comment (santanaNogueira) [reply docs/ios-sdk.md:112]: Sorry, that was a mistake when I copied the "Life Cycle" to below the Usage section. I think change it to "## Livery SDK" is enough. And change all the sub-section of it too, of course.

File: docs/ios-sdk.md (hunk 3 above)
Comment (santanaNogueira) [reply docs/ios-sdk.md:394]: No, error handling is implemented. We can remove it from the docs.

File: docs/ios-sdk.md (hunk 1 above)
Comment (joepadmiraal) [reply docs/ios-sdk.md]: ok

File: docs/ios-sdk.md (hunk 3 above)
Comment (joepadmiraal) [reply docs/ios-sdk.md]: ok

**Pull request merged**